src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -19,16 +19,8 @@ import {
 } from '../../utils/utils';
 
 class HoldConfirmation extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      patron: PatronStore.getState(),
-    };
-  }
-
   componentDidMount() {
-    this.requireUser();
+    // this.requireUser();
```

Comment:
This should probably not be here

Reply:
Thanks. Uncommented that out and working on a new bug I found. For some reason, when I submit an EDD request, the bib is removed from state for the hold confirmation page. Investigating right now.

Reply:
The above issue was resolved here: https://github.com/NYPL-discovery/discovery-front-end/pull/1465/commits/1a3a582c95d1224eba870c42c4ac31627135d19c#diff-cbddc9a9f7f5736d9cfbc45df71a1e59R61